### PR TITLE
chore: split ws server in dev env

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**
We are working on issues related with the WS server that requires to run daphne in a separated process to not block api requests.  @mkanoor will do the same for mac.

### CODE REVIEW

File: tools/docker/docker-compose-dev.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ x-environment:
   - DJANGO_SETTINGS_MODULE=${DJANGO_SETTINGS_MODULE:-aap_eda.settings.development}
   - EDA_ALLOWED_HOSTS=['*']
   - EDA_DEPLOYMENT_TYPE=${EDA_DEPLOYMENT_TYPE:-podman}
-  - EDA_WEBSOCKET_BASE_URL=${EDA_WEBSOCKET_BASE_URL:-ws://eda-api:8000}
+  - EDA_WEBSOCKET_BASE_URL=${EDA_WEBSOCKET_BASE_URL:-ws://eda-ws:8000}
```
Comment (mkanoor) [inline tools/docker/docker-compose-dev.yaml:8]: @Alex-Izquierdo should this be 8001

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose-dev.yaml:8]: No, the process is listening at 8000, 8001 is only for access from the host.

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose-dev.yaml:8]: here we are using the internal dns "eda-ws" it means it makes the request directly to the host. If the dns would be "host.container.internal" then yes, it should be 8001

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose-dev.yaml:8]: yeah, that's why I leave on your hands that part ;)

Comment (Alex-Izquierdo) [reply tools/docker/docker-compose-dev.yaml:8]: Anyway, it is weird, I think podman-machine should support internal resolution, its a very common feature.  Maybe its something that must be enabled manually.

**Review by mkanoor — DISMISSED** [review]

**Review by Dostonbek1 — APPROVED** [review]
lgtm

**Review by mkanoor — APPROVED** [review]

**Pull request merged**